paasta_tools/cli/cmds/validate.py

```diff
@@ -299,16 +322,11 @@ def validate_schema(file_path: str, file_type: str) -> bool:
     :param file_type: what schema type should we validate against
     """
     try:
-        schema = get_schema(file_type)
+        validator = get_schema_validator(file_type)
     except Exception as e:
         print(f"{SCHEMA_ERROR}: {file_type}, error: {e!r}")
         return False
 
-    if schema is None:
-        print(f"{SCHEMA_NOT_FOUND}: {file_path}")
-        return False
```

Comment:
what happens if there's no schema now? will creating the validator thrown an exception which then gets caught by the catch-all handler above?

Reply:
from the test, i assume that's what will happen/what we wanted to happen